DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -995,13 +906,47 @@ public void PromoteDelegateToAdmin_calls_data_service_with_expected_value()
                                 a.IsTrainer == adminRoles.IsTrainer &&
                                 a.IsSupervisor == adminRoles.IsSupervisor
                         ),
-                        userId
+                        1
                     )
-                ).MustHaveHappened();
-                UpdateToExistingAdminAccountMustNotHaveHappened();
+                ).MustNotHaveHappened();
             }
         }
 
+        [Test]
+        public void PromoteDelegateToAdmin_calls_data_service_with_expected_values_if_no_existing_admin()

```

Comment:
Might want to modify this test so that the user has active/inactive admin accounts at different centres, just to ensure we can create multiple admins easily (as that wasn't possible pre-UAR).